fix(design-tokens): fix broken type declaration template

andipants12 opened the pull request:
<!-- Describe your Pull Request -->

## Fix typo in `gatsby.d.ts.ts` theo template
- Changes in `@twilio-paste/design-tokens`
- Template had an extra `{` character which has been removed
- Previously had typed the tokens object as `readonly` as we do not intend for the built object to be [frozen](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze).

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

TheSisb commented on .changeset/curly-wolves-promise.md:
```diff
@@ -0,0 +1,9 @@
+---
+'@twilio-paste/design-tokens': patch
+'@twilio-paste/core': patch
+---
+
+[Design tokens] fix typo in gatsby tokens type declaration template and remove readonly declaration
+
+- Remove extra `{` character
+- Remove typing as a readonly object in the `gatsby.d.ts` temoplate
```
thought: since the Version Packages PR hasn't merged yet, we can just piggy back on that changeset file and not have to announce to the world we broke something.  If possible I'd suggest updating that changeset file to something a little more external consumer friendly like:


"[Design tokens] Add a new generic export featuring additional metadata. This can be used in documentation or software tooling."

And then we can rename it to something like `generic.ts` instead of `gatsby.ts` and our friend with the VSCode extension can use it too.  What do you think?

TheSisb commented on packages/paste-design-tokens/formatters/gatsby.d.ts.ts:
```diff
@@ -59,7 +59,7 @@ export const formatGroupTokensWithTemplate = (
     .join(',\n');
 
-  return `{\n${content}\n};`;
+  return content;
```
praise: good fixes!

andipants12 commented:
@TheSisb thanks for the review! I just made the following changes:
- removed the additional changeset and updated the memo in the persistent one
- renamed the newly added formatter files to be `generic.ts`, `generic.d.ts.ts` accordingly, as well as the build output files.

The output files built now are:
- ~`/dist/tokens.gatsby.js`~    👉 `/dist/tokens.generic.js`
- ~`/dist/tokens.gatsby.d.ts`~  👉   `/dist/tokens.generic.d.ts`
- ~`/dist/themes/dark/tokens.gatsby.js`~     👉 `/dist/themes/dark/tokens.genericjs`
- ~`/dist/themes/dark/tokens.gatsby.d.ts`~     👉 `/dist/themes/dark/tokens.generic.d.ts`

TheSisb commented on packages/paste-design-tokens/formatters/__tests__/__snapshots__/gatsby.d.ts.spec.ts.snap:
```diff
@@ -2,738 +2,736 @@
 
```
issue: we may have to regenerate this snapshot under the new name

zahnster reviewed (approved):
Praise: thanks for the quick turnaround on this! mah types will be happy

loreina reviewed (approved):
note: didn't require a design review, but design-systems-pd was tagged to review since it affects the design-tokens package
